src/index.ts

```diff
@@ -44,67 +44,66 @@ const argv = yargs(process.argv.splice(2))
 	o: {
 		alias: 'open',
 		group: 'General Options:',
-		desc: 'Open in browser',
+		desc: "Open in browser",
 		type: 'boolean',
 		default: false
 	},
 	i: {
 		alias: 'init',
 		group: 'General Options:',
-		desc: 'Init serve project',
+		desc: "Init serve project",
 		type: 'boolean',
 		default: false
 	},
 	e: {
 		alias: 'err-page',
 		group: 'General Options:',
-		desc: 'Path to the file that server will respond with if an Internal Server Error occurs. Priority: arg > ./.500.html > SCRIPT_DIR/assets/500.html',
+		desc: "Path to the file that server will respond with if an Internal Server Error occurs. Priority: arg > ./.500.html > SCRIPT_DIR/assets/500.html",
 		type: 'string',
 		default: null,
 		defaultDescription: '.500.html'
 	},
 	n: {
 		alias: 'not-found-page',
 		group: 'General Options:',
-		desc: 'Path to the file that server will respond with if requested path is not found. Priority: arg > ./.404.html > SCRIPT_DIR/assets/404.html',
+		desc: "Path to the file that server will respond with if requested path is not found. Priority: arg > ./.404.html > SCRIPT_DIR/assets/404.html",
 		type: 'string',
 		default: null,
 		defaultDescription: '.404.html'
 	},
 	r: {
 		alias: 'routes',
 		group: 'General Options:',
-		desc: 'Path to routes map. Priority: arg > ./.routes',
+		desc: "Path to routes map. Priority: arg > ./.routes",
 		type: 'string',
 		default: null,
 		defaultDescription: '.routes'
 	},
 	'sd': {
 		alias: 's-dotfiles',
 		group: 'Static Options:',
-		desc: 'Determines how dotfiles (files or directories that begin with a dot “.”) are treated',
+		desc: "Determines how dotfiles (files or directories that begin with a dot “.”) are treated",
```

Comment:
Why would you change outer quotes instead of the ones that are INSIDE the text?
```suggestion
		desc: 'Determines how dotfiles (files or directories that begin with a dot ".") are treated',
```

Reply:
You may have fixed this one line, but what about the others?